cmd/project.go

```diff
@@ -145,6 +145,21 @@ var projectDeleteCmd = &cobra.Command{
 			checkError(err, "")
 		}
 		fmt.Printf("Deleted project : %v\n", projectName)
+
+		// Get Current Project
+		currProject := project.GetCurrent(client)
+
+		// Check if List returns empty, if so, the currProject is showing old currentProject
+		// In openshift, when the project is deleted, it does not reset the current project in kube config file which is used by odo for current project
+		projects, err := project.List(client)
+		checkError(err, "")
+		if len(projects) != 0 {
+			fmt.Printf("%s has been set as active project\n", currProject)
```

Comment:
can't believe I missed this haha.

`has been set as the active project`

Reply:
Ack :)